pycytominer/annotate.py

```diff
@@ -113,6 +117,7 @@ def annotate(
                 left_on=external_join_left,
                 right_on=external_join_right,
                 how="left",
+                suffixes=[None, "_external"],
```

Comment:
The source of the external metadata can be pretty broad. In my case it's a tabular file that I export from a benchling metadata python package. I choose the name `_external` because that's close to the argument in question `external_metadata`. We could do the full `external_metadata`, but I think that's a bit long and unwieldy.

Reply:
I think `_external` is sufficient